apps/aevm/src/aevm_ae_primops.erl

```diff
@@ -831,7 +831,7 @@ map_call_tolist(_Gas, Data, State) ->
     [MapId] = get_args([word], Data),
     {KeyType, ValType} = aevm_eeevm_maps:map_type(MapId, State),
     {ok, Map} = aevm_eeevm_maps:get_flat_map(MapId, State),
-    List = maps:to_list(Map),
+    List = lists:keysort(1, maps:to_list(Map)),
```

Comment:
Theoretically AEVM is still active, you can post new contracts until the IRIS protocol upgrade, so maybe we should have the same/a similar mechanism with a file containing mappings here?